code/components/citizen-resources-core/src/StateBagComponent.cpp

```diff
@@ -619,10 +620,36 @@ void StateBagComponentImpl::HandlePacket(int source, std::string_view dataRaw)
 
 	if (!bag)
 	{
+#ifdef IS_FXSERVER
+		if (strncmp(bagName.data(), "entity:", 7) == 0)
```

Comment:
This file _should_ be generic: no specific server logic, and ideally no specific parser for `entity:` either. The game interface calls should reflect this.

Reply:
👍